lib/cylc/job_file.py

```diff
@@ -445,7 +447,7 @@ def _write_epilogue(cls, handle, job_conf):
 echo 'JOB SCRIPT EXITING: THIS TASK HANDLES ITS OWN COMPLETION MESSAGING'
 trap '' EXIT
 
-#EOF""")
```

Comment:
Yes, but it is often a good delimiter to indicate the end of file to a human reader.